keps/sig-cli/0008-kustomize.md

```diff
@@ -0,0 +1,219 @@
+---
+kep-number: 8
+title: Kustomize
+authors:
+  - "@pwittrock"
+  - "@monopole"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+reviewers:
+  - "@droot"
+approvers:
+  - "@maciej"
+editor: "@droot"
+creation-date: 2018-05-5
+last-updated: 2018-05-5
+status: implemented
+see-also:
+  - n/a
+replaces:
+  - kinflate
```

Comment:
This is supposed to be a reference to another KEP.

Reply:
No, its just the old name.  I will clarify.